ament_cmake_cppcheck/cmake/ament_cmake_cppcheck_lint_hook.cmake

```diff
@@ -33,6 +33,12 @@ if(_source_files)
     list(APPEND _all_include_dirs ${ament_cmake_cppcheck_ADDITIONAL_INCLUDE_DIRS})
   endif()
 
+  # Language
+  set(_language "")
+  if(DEFINED ament_cmake_cppcheck_LANGUAGE)
+    set(_language ${ament_cmake_cppcheck_LANGUAGE})
+  endif()
```

Comment:
This passes an empty `LANGUAGE` in the `else` case. I would argue it shouldn't pass `LANGUAGE` at all in that case.

Reply:
> and also not print a message in that case

This part of my comment is still pending

Reply:
https://github.com/ament/ament_lint/pull/239/commits/67861398c1f4be21d0429ee3865a3e67633c17f9